packages/postgres-database/src/simcore_postgres_database/models/projects_comments.py

```diff
@@ -25,12 +26,19 @@
             onupdate="CASCADE",
         ),
         index=True,
+        nullable=False,
         doc="project reference for this table",
     ),
     sa.Column(
         "user_id",
         sa.BigInteger,
+        sa.ForeignKey(
+            users.c.id,
+            name="fk_projects_comments_user_id",
+            ondelete="SET NULL",
+        ),
         doc="user who created the comment",
```

Comment:
I would add a comment as : "NOTE that if the user gets deleted, it sets to `null` which should be interpreted as "unknown" user.